### PR TITLE
Av fixtures

### LINKED ISSUE #2264: Implement Cypress tests for Video/Audio
**Is your feature request related to a problem? Please describe.**
Currently, we have no Cypress tests for the video/audio components.

**Describe the solution you'd like**
Implement Cypress tests for Video/Audio components.

**Testing notes**
Cypress tests need to be written to test functionality of iframe/video & audio player. The functionality tests will include making sure the video player appears and that it plays, as expected. Before writing other tests, a tester will need to sit with a developer to go through current Snapshot tests to make sure there's no duplication between unit/Cypress tests. This will just be for the canonical article as AMP currently is unavailable. These tests will be written using the Global video variant.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #2264 

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

* Adds two new scenarios
* The TEST asset containing multiple AV scenarios
* The LIVE asset containing one video

**Code changes:**

* Adds two new files to data
* Updates tests to take new files into account
* Updates readme to describe new files

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: data/README.md [hunk 1]
```diff
@@ -39,7 +39,12 @@ The assets sequentially numbered are for local testing, those which aren't are e
 | 28       | Home: Persian | Headline, Image, Text (5 pars), Image (wide), Text (3 pars), Image (wide, caption), Text (3 pars) | One tag | Prod and Test |
 | 29       | Home: News | Headline (Image), Subheadline (Video), Text (1 par), Subheadline (Image), Text (1 par)| | |
 
-## Video test assets
+## Audio/Video test assets
+
+| ID | Description |
+| -----------: | ------------- |
+| c3wmq4d1y3wo | TEST asset containing multiple AV scenarios |
+| cd4117egk3go | PROD asset containing one live environment video |
```
Comment (jamesdonoh) [inline data/README.md:47]: Does this render the section below redundant?

Comment (jamesdonoh) [reply data/README.md:47]: Is there some way we could indicate that this asset is also (now) available as a local fixture?

Comment (jimjohnsonrollings) [reply data/README.md:47]: Yes.  Will remove the below section and update the intro to be clearer that they are available locally.

**Review by twinlensreflex — APPROVED** [review]
looks good to me!

**Review by oluoluoxenfree — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**